crates/ruff/src/rules/flake8_bandit/helpers.rs

```diff
@@ -10,10 +10,10 @@ static PASSWORD_CANDIDATE_REGEX: Lazy<Regex> = Lazy::new(|| {
 
 pub fn string_literal(expr: &Expr) -> Option<&str> {
     match &expr.node {
-        ExprKind::Constant {
+        ExprKind::Constant(ast::ExprConstant {
             value: Constant::Str(string),
             ..
-        } => Some(string),
+        }) => Some(string),
```

Comment:
You may worry about this patterns by looking more verbose than before. Enum definition is changed to be using struct of enums. This is transitional state and it can be written like this later:
```suggestion
        ExprKind::Constant(constant) => Some(constant.value)
```

By allowing adding methods to `ast::ExprConstant` structs unlike before, it will be even better in the future.

Reply:
I like it! The possibility for adding methods to `ExprConstant`, and that we can now pass specific `Stmt` instead of passing `Stmt` and than unwrapping a specific kind makes the API much more ergonomic